training_config/dependency_parser_spacy.jsonnet

```diff
@@ -0,0 +1,76 @@
+{
+    "dataset_reader":{
+      "type":"universal_dependencies",
+      "token_indexers": {
+        "tokens": {
+          "type": "spacy"
+        },
+      },
+
+      "tokenizer": {
+        "type": "word",
+        "word_splitter": {
+          "type": "spacy",
+          "split_on_spaces": true,
+          "keep_spacy_tokens": true,
+          "pos_tags": true,
+        },
+      },
+    },
+    "train_data_path": "/Users/markn/allen_ai/data/dependency-parsing/en-ud/en_ewt-ud-train.conllu",
```

Comment:
?

Reply:
removed, this was accidentally checked in 😓